### PR TITLE
Commission details

### PR DESCRIPTION
**Pull request by mjankowski**
Formats to use "Semantic Line Feeds" style.

Clarifies (hopefully) what were already existing policies around how we calculate commission, and that claims to commission ends when employment ends.

### CODE REVIEW

File: benefits/commission.md [hunk 1]
```diff
@@ -1,29 +1,76 @@
...
+This may be split amongst multiple sales-people.
+A project does not have to have a sales-person.  
+**Time cutoff:** Commission is earned only on revenue
+created by work performed in the first 12 months
+after a project begins.  
+**Payment:** Commission is earned only on actual payments received,
+and not on generated billings which have not yet been paid.
+**Rate floor:** Commission is paid only on projects
+with an effective rate at or above
+$5,000/person/week.
```
Comment (croaky) [inline benefits/commission.md]: "Time cutoff" and "Payment" now say "is earned", which seems like the right verb. Make "Rate floor" consistent?

How about separately explaining the cycle for paying earned commissions? Accountant calculates it after month (e.g. January) ends, goes into the next payroll (e.g. mid-February).

Comment (croaky) [inline benefits/commission.md]: Nevermind, I see that comes later.

File: benefits/commission.md [hunk 2]
```diff
@@ -1,29 +1,76 @@
...
+based on that client,
+such that the Company will decide
+to enter into an agreement with that client.
 
-For purposes of this policy the term “procured” means the Employee’s introduction of the Company to an appropriate representative of a potential client, for the purpose of the Company entering into an agreement to sell services or goods to that potential client.
+For purposes of this policy
+the term “procured” means the Employee’s introduction of the Company
+to an appropriate representative of a potential client,
+for the purpose of the Company entering into an agreement
+to sell services or goods to that potential client.
```
Comment (croaky) [inline benefits/commission.md]: Huh, I don't remember much of any of lines 21-39 before. Seems like those details are now better explained in the cookbook?

Interesting that we really focused on the introduction of the client here, and not the processing (sales) of a lead.

File: benefits/commission.md [hunk 3]
```diff
@@ -1,29 +1,76 @@
...
 
 ## Payments
```
Comment (croaky) [inline benefits/commission.md]: Maybe "Payouts" to match the word used most frequently in this section, and to distinguish from the client paying us?

File: benefits/commission.md [hunk 4]
```diff
@@ -1,29 +1,76 @@
...
 
-All employees can take advantage of this benefit upon starting with thoughtbot.
-Commission is no longer paid to you when your employment with the company ends.
+All employees can take advantage of this benefit
+upon starting with thoughtbot.
+Commission is no longer paid to you
+when your employment with the company ends.
+This includes any commission which would have been due
+on generated billings when employment ends,
+but which have not yet been paid.
```
Comment (croaky) [inline benefits/commission.md]: The part about generated bills here seems redundant to lines 15-16 and seems like it should be removed.

The part that seems like it should be clarified is that when the accountant calculates commissions for a month (e.g. January), they will apply the commission split for each payment received per the commission policy on that day.

Lines 65-66 also seems factually incorrect. We *will* pay any due commissions to people who leave once the accountant has calculated, it's just that the commission policy changes the day after their last day of employment.

Would an example be most clear?

```
Commission policy for Client from January 1-January 15: 5% to Employee Edgar.
Payment of $10,000 from Client comes in January 10.
Employee Edgar's last day at the company is January 15.
Commission policy for Client from January 15-onward: 0% to all current and former employees.
Payment of $10,000 from Client comes in January 20.
Accountant calculates commissions sometime in early February.
Company pays Employee Edgar for $500 for commission on
the one payment from Client in January before their last day as an employee.
```

**Comment by mjankowski** [comment]
Made the words a little more consistent in each section.

**Comment by mjankowski** [comment]
Moved PR here: https://github.com/thoughtbot/cookbook/pull/100

**Pull request closed without merging**